### PR TITLE
Make Dockerfile templated, reducing the default configuration

### PR DESCRIPTION
**Pull request by saez0pub**
Using default(omit) in docker create.yml, we have less configuration in
mlecule.yml.
The Dockerfile was not explicit when we want to build image because of
the "FROM centos:7".

### CODE REVIEW

File: molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml [hunk 1]
```diff
@@ -8,21 +8,30 @@
     molecule_scenario_directory: "{{ lookup('env','MOLECULE_SCENARIO_DIRECTORY') }}"
     molecule_yml: "{{ lookup('file', molecule_file) | from_yaml }}"
   tasks:
+    - name: Create Dockerfiles from image names
+      template:
+        src: "{{ molecule_scenario_directory }}/Dockerfile-template"
```
Comment (retr0h) [inline molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml]: I don't see this template existing in this PR.

File: molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml [hunk 2]
```diff
@@ -7,9 +7,9 @@ lint:
   name: {{ cookiecutter.lint_name }}
 platforms:
   - name: instance-1
-    image: molecule_local/centos:7
-    command: sleep infinity
-    dockerfile: Dockerfile
+    image: centos:7
+    #command: sleep infinity
+    #dockerfile: Dockerfile
```
Comment (retr0h) [inline molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml]: We can remove the commented lines.

**Comment by saez0pub** [comment]
Done, I had fogotten the git add :)
I also removed the comments.
Maybe  it would be better if the generated template is placed under the .molecule/ directory.

**Comment by retr0h** [comment]
Thanks @saez0pub 🎸

**Pull request merged**